src/components/panel/panel.spec.js

```diff
@@ -105,6 +108,62 @@ describe('$mdPanel', function() {
     expect(PANEL_EL).not.toExist();
   });
 
+  it('should call onDomAdded if provided when adding the panel to the DOM', function() {
+    var onDomAdded = jasmine.createSpy();
+    var animationHookConfig = angular.extend({'onDomAdded': onDomAdded}, DEFAULT_CONFIG);
```

Comment:
This variable name does not explain what is going in it. I think just plain `config` is better in this case.